packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -105,6 +77,28 @@ const makeOuterKit = inner => {
  * @property {(oldDebt: Amount, oldCollateral: Amount, vaultId: VaultId) => void} updateVaultPriority
  */
 
+/**
+ * @typedef {Readonly<{
```

Comment:
I put these in here but stopped short of using them because they're available in closure scope and it doesn't make much sense to change them before factoring out `makeMethods` or whatever we call it. I may have time for that before Monday if it's worthwhile, though I'd prefer to land the non-state parts of this PR first.